src/info/langs/mod.rs

```diff
@@ -31,7 +31,7 @@ fn get_language_distribution(languages: &tokei::Languages) -> Option<HashMap<Lan
     let mut language_distribution = HashMap::new();
 
     for (language_name, language) in languages.iter() {
-        let mut code = language.code;
+        let mut code = language::compile_tokei_loc(language_name, language);
```

Comment:
```suggestion
        let mut loc = language::loc(language_name, language);
```
minor: it may be confusing because from my understanding the "compile" part (tera template -> rust) is done at build time and not at run time.

Reply:
>from my understanding the "compile" part

Oh, yeah, maybe "compile" wasn't the best word to use. Simple `loc` is much more clear :+1: